utils/utils.go

```diff
@@ -54,3 +57,37 @@ func Prctl(option int, arg2, arg3, arg4, arg5 uintptr) (err error) {
 func StatusToExitCode(status int) int {
 	return ((status) & 0xff00) >> 8
 }
+
+// RunUnderSystemdScope adds the specified pid to a systemd scope
+func RunUnderSystemdScope(pid int, slice string, unitName string) error {
+	var properties []systemdDbus.Property
+	conn, err := systemdDbus.New()
+	if err != nil {
+		return err
+	}
+	properties = append(properties, systemdDbus.PropSlice(slice))
+	properties = append(properties, newProp("PIDs", []uint32{uint32(pid)}))
+	properties = append(properties, newProp("Delegate", true))
+	properties = append(properties, newProp("DefaultDependencies", false))
+	if _, err := conn.StartTransientUnit(unitName, "replace", properties, nil); err != nil && !isUnitExists(err) {
```

Comment:
Removed that additional check so we return the err. We could introduce it later if we need it when I finish the integration.